File: spologp_v/lista2sala-C/questao4.c
#include<stdio.h>
#include<stdbool.h>

int main()
{
    int A, B, C;
    bool triangulo;

    printf("Digite o valor do lado A:");
    scanf("%i", &A);
    printf("Digite o valor do lado B:");
    scanf("%i", &B);
    printf("Digite o valor do lado C:");
    scanf("%i", &C);

if((A+B>C) && (A+C>B) && (B+C>A))
    triangulo = true;

else{
    triangulo = false;
}

if(triangulo == true){
    if((A==B) && (A==C)){
    printf("Esse é um triângulo equilátero");
    }
    else if((A==B) || (A==C) || (B==C)){
        printf("Esse é um triângulo isósceles");
    }

    else{
        printf("Esse é um triângulo escaleno");
    }
}

if (triangulo == false){
    printf("Esses valores não correspondem a um triângulo");
}

return 0;
}

File: spologp_v/lista2sala-python/questao4.py
ladoA = int(input("Digite o valor do lado A:"))
ladoB = int(input("Digite o valor do lado B:"))
ladoC = int(input("Digite o valor do lado C:"))
triangulo = True

if(ladoA<ladoB+ladoC) and (ladoB<ladoA+ladoC) and (ladoC<ladoA+ladoB):
    triangulo = True
else:
    triangulo = False

if triangulo==True:
    if(ladoA==ladoB) and (ladoB==ladoC):
        print("Esse é um triângulo equilátero.")
    elif(ladoA==ladoB) or (ladoB==ladoC) or (ladoA==ladoC):
        print("Esse é um triângulo isósceles.")
    else:
        print("Esse é um triângulo escaleno.")

if triangulo==False:
    print("Esse não é um triângulo.")